spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/optimizer/RewriteMergeInto.scala

```diff
@@ -216,8 +215,10 @@ case class RewriteMergeInto(spark: SparkSession) extends Rule[LogicalPlan] with
   }
 
   private def isCardinalityCheckEnabled(table: Table): Boolean = {
-    PropertyUtil.propertyAsBoolean(table.properties(),
-      MERGE_WRITE_CARDINALITY_CHECK, MERGE_WRITE_CARDINALITY_CHECK_DEFAULT)
+    PropertyUtil.propertyAsBoolean(
+      table.properties(),
+      MERGE_CARDINALITY_CHECK_ENABLED,
+      MERGE_CARDINALITY_CHECK_ENABLED_DEFAULT)
```

Comment:
This has a few places that don't really need to be refomatted, like this. It should be okay since these are new and it is unlikely that anyone is cherry-picking these commits, but I'd prefer to avoid them in the future.